README.md

```diff
@@ -320,98 +794,368 @@ And configure your remote endpoint:
+```javascript
+import { render } from 'hops';
+import { action, observable } from 'mobx';
+import { inject, observer } from 'mobx-react';
+import React from 'react';
+
+const theme = observable(
+  {
+    color: 'red',
+    setColor(color) {
+      this.color = color;
+    },
+  },
+  { setColor: action }
+);
+
+const MyApp = inject('theme')(
+  observer(props => (
+    <h1
+      style={{ color: props.theme.color }}
+      onClick={() => props.theme.setColor('green')}
+    >
+      hello world
+    </h1>
+  ))
+);
+
+export default render(<MyApp />, { mobx: { stores: { theme } } });
+```
+
+### Presets
+
+Presets are being used to provide reusable / shareable [settings](#settings). Optionally they can also provide and configure their own mixins.
+
+A preset can either be written as JavaScript, JSON or yaml file (`preset.{js,json,yml}`) or can be specified in the `"preset"` field in a `package.json`.
+
+#### Example: Preset containing shared settings
+
+Firstly we need to create a new folder (or a npm package) for our preset:
+
+```shell
+mkdir my-awesome-preset
+```
+
+Then we create a `preset.js` file in this folder and specify some settings and their values that we want to share:
+
+**`preset.js`**
+
+```javascript
+module.exports = {
+  browsers: ['IE11', '> 0.5%'],
+};
+```
+
+Now we need to make this preset known to your application. Read the [activating presets](#activating-presets) section above to see how you can activate a preset.
+
+#### Example: Preset containing custom mixins
+
+Usually when you create a mixin and want to share it with others, you would create an npm package that contains the mixin and also a preset that specifies all [settings](#settings) that your mixin uses and defines some sane defaults for it.
+
+Firstly we need to create a new folder (or an npm package) for our preset:
+
+```shell
+mkdir my-awesome-preset
+```
+
+Then we create a `preset.js` file in this folder and specify your included mixins via the `"mixins"` settings:
+
+**`preset.js`**
+
+```javascript
+module.exports = {
+  mixins: [__dirname],
+};
+```
+
+This will instruct Hops to [look for mixins](#mixins) inside this directory, so you should place a `mixin.{core,runtime,browser,server}.js` file there too containg your mixin code.
+
+#### Example: Shareable preset to bundle and configure other presets
+
+Presets can also contain and configure other presets. This is useful if, for example, you want to provide a company-wide selection of Hops presets inside a single dependency.
+
+First we need to create a new npm package which contains the following `preset.js` file:
+
+**`preset.js`**
+
+```javascript
+module.exports = {
+  graphqlUri: 'https://my-company.com/graphql',
+  presets: ['hops', 'hops-graphql', 'hops-styled-components'],
+};
+```
+
+And then this new package / preset should have all the presets that it references as its dependencies, so that your users don't have to install them themselves.
+
```

Comment:
Would this be the right place to mention you can use any other hops project as a preset?